comp/logs/agent/config/config_keys_test.go

```diff
@@ -80,8 +86,9 @@ func TestGetAdditionalEndpoints(t *testing.T) {
 		}]`
 	configMock.SetWithoutSource("logs_config.additional_endpoints", jsonString)
 
-	endpoints := l.getAdditionalEndpoints()
+	endpoints, source := l.getAdditionalEndpoints()
```

Comment:
Nit: use `path` instead of `source` to be consistent with the other tests written above.